synapse/cortex.py

```diff
@@ -1951,14 +1953,16 @@ async def _confirmStormPkg(self, pkgdef):
         svciden = pkgdef.get('svciden')
 
         if onload is not None:
-            self.getStormQuery(onload)
+            await self.getStormQuery(onload)
```

Comment:
Maybe we should make the sleep(0) happen inside and only on a cache miss?

Reply:
Updated